packages/connectors/src/metaMask.ts

```diff
@@ -186,8 +235,13 @@ export function metaMask(parameters: MetaMaskParameters = {}) {
               return [chain.id, url]
             }),
           ),
-          dappMetadata: parameters.dappMetadata ?? { name: 'wagmi' },
-          useDeeplink: parameters.useDeeplink ?? true,
+          dappMetadata: parameters.dappMetadata ?? {
+            url: `${window.location.protocol}//${window.location.host}`,
+          },
+          useDeeplink: true,
+          injectProvider: false,
+          forceInjectProvider: false,
+          forceDeleteProvider: false,
```

Comment:
What were the defaults internally before? Just want to make sure changes these isn't too disruptive for folks. Probably should allow folks to override these manually so it's not a breaking change and remove that ability in the next major version. (Very unlikely anyone will override manually since people just use defaults, but want to make sure semver is respected.)

Reply:
Is this PR dependent on that one? Haven't had a chance to wrap up our thinking on that quite yet.